### DISCUSSION TITLE
Longtable change spacing between table and text before/after

### DISCUSSION THREAD

**Discussion by briha** [lvjr/tabularray#500] [Q&A]
A normal table has zero spacing between the text before and itself, a long table has a non-zero spacing.
How do I control the length of said spacing (a long table with `\SetTblrTemplate{head}{empty}`, i.e., no header)?

Same question for the footer, if it is empty (because the table fits on one table): How do I control the margin there?

**Comment by sgolovan** [comment]
Look at different `*sep` options at page 39 (section 4.1) of the manual.

**Reply by briha** [reply]
Completely missed that, that does the trick though. Thanks!
